command/src/main/java/com/jagrosh/jdautilities/command/impl/CommandClientImpl.java

```diff
@@ -666,7 +666,7 @@ private MessageParts getParts(MessageReceivedEvent event) {
         if(prefix.equals(DEFAULT_PREFIX) || (altprefix != null && altprefix.equals(DEFAULT_PREFIX))) {
             if(rawContent.startsWith("<@"+ event.getJDA().getSelfUser().getId()+">") ||
                     rawContent.startsWith("<@!"+ event.getJDA().getSelfUser().getId()+">")) {
-                final int prefixLength = rawContent.indexOf('>') + 1;
+                final int prefixLength = rawContent.indexOf('>') + 2;
```

Comment:
I suggest you add a comment saying why the + 2 is needed

Reply:
Ok will add comment asap

Reply:
Sorry for the wait, I have added comments with the commit b5402a9d1f72b8ec9afd1be519cd089017f8e635. 
I hope that is explicit, English is not my native language